docs/installation.rst

```diff
@@ -153,7 +153,7 @@ run tasks:
 
 .. code-block:: bash
 
-    airflow db init
+    airflow initdb  
```

Comment:
The doc in master branch is reflecting the latest code/doc we have in 2.0.*.

In 2.0.*, the CLI has been refactored. `airflow initdb` will no longer be there, and we have `airflow db init.`

Reply:
You can refer to this documentation: https://github.com/apache/airflow/blob/master/UPGRADING_TO_2.0.md#airflow-cli-changes-in-20